### ISSUE TITLE
When alarm is triggert rules ate unchangable

### ISSUE THREAD

**Issue by JoshyJS** [stuartpittaway/diyBMSv4Code#128]
Condition: CAN and data cables are connected.
Battery is a 16S, min V when empty 40000mV.
Default setting of the SW is 33600mV.
Alarm is immediately triggert when starting the BMS.
When alarm is triggert the values under rules are not adjustable.
"Failed to change settings"

**Comment by stuartpittaway** [comment]
I'd have to investigate why this is occurring, have you tried refreshing the page in the web browser? Or try another web browser.

**Comment by JoshyJS** [comment]
Yes I did. As long as an alarm is triggert no changes in rules are possible.
Please read also the explanation on  OEM.
